File: app/src/main/java/com/example/connivingdog/numberguess/SimPrefix.java
package com.example.connivingdog.numberguess;

public class SimPrefix {
    private String simCard;
    private String simPref;

    public SimPrefix(String simCard, String simPref) {
        this.simCard = simCard;
        this.simPref = simPref;
    }

    public SimPrefix() {
    }

    public String getSimCard() {
        return simCard;
    }

    public void setSimCard(String simCard) {
        this.simCard = simCard;
    }

    public String getSimPref() {
        return simPref;
    }

    public void setSimPref(String simPref) {
        this.simPref = simPref;
    }
}

File: app/src/main/java/com/example/connivingdog/numberguess/SimPrefix.kt
package com.example.connivingdog.numberguess

class SimPrefix {
    var simCard: String? = null // properties,
    var simPref: String? = null // also acts as the getter and setter at the same time

    constructor(simCard: String, simPref: String) { //constructor with parameters
        this.simCard = simCard
        this.simPref = simPref
    }

    constructor() {} // empty constructor
}
